critical_demand.py

```diff
@@ -346,6 +348,16 @@ def run_simulation(n_days=n_days, case=case):
 
     results = solph.processing.results(model)
 
+    df_results = df_costs.copy()
+    df_results["capacity"] = 0
+    df_results["total_flow"] = 0
+    df_results["cash_flow"] = 0
+    df_results["cash_flow"] = 0
+
+    project_lifetime = 20
+    wacc = 0.06
+    CRF = annuity(1, project_lifetime, wacc)
```

Comment:
Do we need to add annuity here? to calculate CRF, we need to apply this equ.:  
CRF=(wacc ⋅ (1+wacc)^T)/(〖(1+wacc)〗^T-1)

Reply:
Annuity with a capex of 1 is equal to CRF : https://oemof-solph.readthedocs.io/en/v0.2.3/api/oemof.tools.html#oemof.tools.economics.annuity